scripts/generate-iscsi-iqn

```diff
@@ -7,6 +7,9 @@ set -e
 export FIRSTBOOT_DATA_DIR=/etc/firstboot.d/data
 export XENSOURCE_INVENTORY=/etc/xensource-inventory
 
+UPGRADE="false"
+[ -r ${FIRSTBOOT_DATA_DIR}/host.conf ] && . ${FIRSTBOOT_DATA_DIR}/host.conf
```

Comment:
Using `-r` instead of the usual `-e` seems to be useful as the executable flag in the file doesn't matter anyway. From the manpage:


> `.  filename [arguments]`
> `source filename [arguments]`
> Read and execute commands from filename in the current shell environment and return the exit status of the last command executed from filename.  If  filename  does     
> not contain a slash, filenames in PATH are used to find the directory containing filename.  **The file searched for in PATH need not be executable.**